Annotato/Annotato/Adapters/AnnotatoAuthAdapter.swift

```diff
@@ -0,0 +1,9 @@
+import AnnotatoSharedLibrary
+
+protocol AnnotatoAuthAdapter {
+    var delegate: AnnotatoAuthDelegate? { get set }
+    var annotatoAuthUser: AnnotatoUser? { get }
```

Comment:
```suggestion
    var currentUser: AnnotatoUser? { get }
```
Will this better convey the intent?